src/harrastuspassi/harrastuspassi/models.py

```diff
@@ -223,7 +226,7 @@ class HobbyEvent(ExternalDataModel, TimestampedModel):
         (7, _('Sunday')),
     )
     hobby = models.ForeignKey(Hobby, related_name='events', verbose_name=_('Hobby'),
-                              null=False, on_delete=models.CASCADE)
+                              blank=True, null=True, on_delete=models.CASCADE)
```

Comment:
otherwise it is impossible to create orphaned events, i.e. the HobbyEvents that still have no Hobby, while the importer is running.